src/dtgtk/range.c

```diff
@@ -1738,6 +1744,59 @@ gchar *dtgtk_range_select_get_bounds_pretty(GtkDarktableRangeSelect *range)
   return txt;
 }
 
+gchar *dtgtk_range_select_get_rating_bounds_pretty(GtkDarktableRangeSelect *range)
```

Comment:
I think that this can go in libs/filters/rating.c as as static func...

Reply:
moved